textproc/dblatex/Portfile

```diff
@@ -125,10 +122,9 @@ proc dblatex.mktexlsr {} {
     if {![variant_isset mactex]} {
         texlive.mktexlsr
     } else {
-        global dblatex.mactex_bin
-        system "${dblatex.mactex_bin}/mktexlsr"
-        if [file exists "${dblatex.mactex_bin}/mtxrun"] {
-            system "${dblatex.mactex_bin}/mtxrun --generate"
+        system "mktexlsr"
```

Comment:
I don't particularly like this since you don't know which `mktexlsr` will be run here (the one from MacPorts or the one from external distribution).

Reply:
OK, will upload a different solution.

Reply:
I've re-introduced `dblatex.mactex_bin` (but deriving it from the path).